pkgs.nix

```diff
@@ -1,5 +1,13 @@
-import (
-  # let rev = "60cce7e5e1fdf62421ef6d4184ee399b46209366"; in
-  let rev = "04a48a2fcd2370f4e8a8c9fd3394531ccdb30f1e"; in
-  fetchTarball "https://github.com/MatrixAI/nixpkgs/archive/${rev}.tar.gz"
-)
+#import (
+#  # let rev = "60cce7e5e1fdf62421ef6d4184ee399b46209366"; in
+#  # let rev = "04a48a2fcd2370f4e8a8c9fd3394531ccdb30f1e"; in
+#  let rev = "53caacaf56640d04180775aee016d2f16d6f083c"; in
+#  fetchTarball "https://github.com/MatrixAI/nixpkgs/archive/${rev}.tar.gz"
+#)
+
+# Pinning to an overlay with a patch for vercel/pkg
+import (builtins.fetchGit {
+  url = "git@gitlab.com:MatrixAI/Engineering/nixpkgs-overlay.git";
+  rev = "958ab694978371a8d73c328cb515df85c7150960";
```

Comment:
No need for this, just use the branch name below is sufficient. Plus I've rebased and squashed on the branch.